tap_chargebee/streams/subscriptions.py

```diff
@@ -15,3 +15,9 @@ class SubscriptionsStream(BaseChargebeeStream):
 
     def get_url(self):
         return 'https://{}/api/v2/subscriptions'.format(self.config.get('full_site'))
+
+    def get_params(self, params):
+        params = super().get_params(params)
```

Comment:
I think the code is not entering here because the `get_params` base function is in `ChargebeeClient` but the streams inherit from `BaseChargebeeStream`, at least is not working for me locally

Reply:
Fixed! Thank you!